### PR TITLE
Update architecture doc (Issue #806)

### PR DESCRIPTION
**Pull request by simanadler**
Added a short description of what the plotter contains, and that it generates the blueprints.

### CODE REVIEW

File: site/docs/concepts/architecture.md [hunk 1]
```diff
@@ -18,9 +18,8 @@ Modules can provide read/write access or produce implicit copies that serve as l
...
-As data assets may reside in different systems the blueprints are compiled in a `Plotter` CRD (6) that specifies which blueprints have to be executed in which cluster.
+It uses this information to check with external systems (4) if access and/or copy is allowed
+and whether restrictive policies such as masking or hashing have to be applied. It generates a `Plotter` CRD (6) based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.  The plotter contains the modules required and all the information required to deploy them on the cluster where they will run, as well as the flow of data between the asset and the workload through the chosen modules.  The plotter generates a `Blueprint` CRD (7) per cluster, which contains all the information required to deploy the relevant modules for that cluster.
```
Comment (ronenkat) [inline site/docs/concepts/architecture.md:22]: CRD --> custom resource (need to change in all the doc as well)

Comment (simanadler) [reply site/docs/concepts/architecture.md]: Changed in all the site doc pages.

File: site/docs/concepts/architecture.md [hunk 2]
```diff
@@ -17,10 +17,9 @@ Modules can provide read/write access or produce implicit copies that serve as l
...
+It uses this information to check with external systems (4) if access and/or copy is allowed
+and whether restrictive policies such as masking or hashing have to be applied. It generates a `Plotter` custom resource (6) based on on the policy decisions received via the connectors and chooses the modules (5) which are best fit for the requirements that the user specified regarding the access protocol and availability.  The plotter contains the modules required and all the information required to deploy them on the cluster where they will run, as well as the flow of data between the asset and the workload through the chosen modules.  The plotter generates a `Blueprint` custom resource (7) per cluster, which contains all the information required to deploy the relevant modules for that cluster.
```
Comment (froesef) [inline site/docs/concepts/architecture.md:22]: You can write `to deploy them on the cluster whee they will run, as well as the asset details and the flow of data between the assets and the workload through the chosen modules.` (to specifically also write that the assets are declared separately in the plotter)

Also:
`The plotter controller generates a 'Blueprint' custom resource`  (the plotter is just a struct)

**Review by froesef — APPROVED** [review]
Just the one comment I left above. Otherwise LGTM.

**Pull request merged**